scripts/create_baseline_stubs.py

```diff
@@ -62,15 +62,16 @@ def run_isort(stub_dir: str) -> None:
 
 def create_metadata(stub_dir: str, version: str) -> None:
     """Create a METADATA.toml file."""
-    m = re.match(r"[0-9]+.[0-9]+", version)
-    if m is None:
+    match = re.match(r"[0-9]+.[0-9]+", version)
+    if match is None:
         sys.exit(f"Error: Cannot parse version number: {version}")
-    fnam = os.path.join(stub_dir, "METADATA.toml")
-    version = m.group(0)
-    assert not os.path.exists(fnam)
-    print(f"Writing {fnam}")
-    with open(fnam, "w") as f:
-        f.write(
+    filename = os.path.join(stub_dir, "METADATA.toml")
+    version = match.group(0)
+    if os.path.exists(filename):
+        return
```

Comment:
Just to check: this is the only "substantive" change in this function, correct? The others are all readability improvements?

(In general, I don't mind readability improvements, but it's easier to review if they're in separate PRs -- it's a bit difficult to see what the substantive changes are here as opposed to the readability improvements :)

Reply:
Indeed. The only "substantive" change is to silently pass instead of erroring out if METADATA.tom already exists.
The only other changes in this function are non-shortened variable names for readability.